script.js

```diff
@@ -198,3 +200,21 @@ function formValidate(event) {
 }
 
 form.addEventListener('submit', formValidate);
+
+function storeData() {
+  const user = {
+    UserName: userName.value,
+    UserEmail: email.value,
+    UserText: message.value,
+  };
+  localStorage.setItem('user', JSON.stringify(user));
+}
+
+form.addEventListener('submit', storeData);
+
+const data = JSON.parse(localStorage.getItem('user'));
+if (data) {
+  userName.value = data.UserName;
+  email.value = data.UserEmail;
+  message.value = data.UserText;
+}
```

Comment:
- Hey, For some reason, I do not see any data I add to the form inputs saved in the localStorage without clicking the submit button, and the textarea value disappears after submitting the form, leaving only the email and name input auto-filled. Please, try fixing this so that the form data are not lost on reloading without submitting and after submitting the form.